.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build libraries
+on: [push, workflow_dispatch]
+
+jobs:
+    build:
+        name: Build libraries
+        runs-on: macos-latest
```

Comment:
why on mac? also I'd use https://github.com/gradle/gradle-build-action for some dep caching

Reply:
Its needed to build xcframeworks for ios